docs/concepts/components/plugins/index.md

```diff
@@ -0,0 +1,17 @@
+# Components
+
+```mdx-code-block
+import { useCurrentSidebarCategory } from '@docusaurus/theme-common';
+import DocCardList from '@theme/DocCardList';
+```
+
```

Comment:
I guess it would be helpful to add 2-3 sentences what plugins are and why they are useful.

Reply:
True, added.